data/remnant/remnant 2 side missions.txt

```diff
@@ -400,3 +400,235 @@ mission "Remnant: Teciimach Deployment"
+			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
+			`	Ruach's gestures take on a more precise cadence. "Taely tells me that you have spotted a strange 'ghost' in Caeculus that bears a resemblance to ancestral ships. Could you take me to do a scan of it please?"`
+			choice
+				`	"Okay."`
+					accept
+				`	"Sorry, not interested."`
+					decline
+	on enter "Caeculus"
+		dialog
+			`Ruach pores over the scanning arrays before reporting that the subject is not present. She suggests searching nearby systems to locate it.`
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system "Levana"
+		ship "Modified Dromedary Specter" "???"
+		dialog "This ship seems a bit more solid than it did when you first saw it, but it does not seem to be quite present."
+	on complete
+		payment 50000
+		conversation
+			`The <ship> hasn't even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple of spots down to stare up at the gills of an Albatross. A few moments later, Taely drops out of them, greeting Ruach warmly. By the time you join them, Ruach has a hologram of the spectral ship floating in the air. "It is lacking the confirmation codes or identification info, but I am confident that this is it," muses Ruach as you arrive.`
+			`	"Sorry for leaving you behind, <first>, but this is just such an exciting discovery. This ship you found matches the profile of the Shattered Light - a large city ship that was part of the Exodus but never arrived on this side of the wormhole. There are so many questions to be answered! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`
+			
+
+
+mission "Remnant: Shattered Light 3"
+	name "Ghost Touch"
+	description "Help the Remnant experiment on the Shattered Light."
+	blocked "As you approach the starport your commlink chimes with a message from Ruach requesting that you make space for 13 people as well as 19 tons of cargo."
+	cargo "equipment" 19
+	passengers 13
+	source "Viminal"
+	waypoint "Coluber"
+	to offer
+		has "Remnant: Shattered Light 2: done"
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system "Lucina"
+		ship "Modified Dromedary Specter" "???"
+		conversation
+			action
+				set "remnant: scanned specter"
+			`It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`
```

Comment:
This conversation could still use with a bit more meat.

Reply:
I believe I originally made it this short because dialog boxes look bad when they get more than a sentence or two in them; and conversation panels feel overbearingly huge and a break in flow when used in space. I think it might be in part because a dialog box is right there in the center, where the player's focus already is while in flight, since that's where their ship is; whereas the conversation box is over on the side. Sometimes *way* over to the side, depending on monitor resolution.

Reply:
Yeah, I get why it was short as a dialog, but now as a conversation (since we need the `action` node) we should use some of the room we have.

Reply:
![FKodLbg](https://user-images.githubusercontent.com/32169904/188284932-7e04127a-9f0a-4a96-bfdd-46b63a7ad615.png)

Reply:
I'm currently trying to write this expanded dialog.

For the future, though, could we get `action` stuff enabled for inside dialogs?

Reply:
Probably not. The more appropriate change would be https://github.com/endless-sky/endless-sky/issues/5248,

Reply:
I'm pretty sure there's an issue asking for it to be possible to perform game actions on NPC events, so it can just go outside the dialog/conversation.

Reply:
Well, we needed it here. Chances are good we'll need it again in the future, one way or another. It'd be nice if dialog boxes were more useful, anyway.

Reply:
New text added, so marking this resolved. Anything further here should probably be comments on that expanded text.